### PR TITLE
Issue 256

### PR DESCRIPTION
**Pull request by sonoct**

### CODE REVIEW

**Comment by mushinlogit** [comment]
unfetter-discover/unfetter#256

File: src/app/settings/stix-objects/attack-patterns/attack-pattern-list/attack-pattern-list.component.spec.ts [hunk 1]
```diff
@@ -1,3 +1,34 @@
+<<<<<<< HEAD
```
Comment (mushinlogit) [inline src/app/settings/stix-objects/attack-patterns/attack-pattern-list/attack-pattern-list.component.spec.ts]: Can you resolve these commit merge markers

File: src/app/settings/stix-objects/course-of-actions/course-of-action/course-of-action.component.spec.ts [hunk 2]
```diff
@@ -1,31 +1,53 @@
 import { NO_ERRORS_SCHEMA } from '@angular/core';
 import { inject, async, TestBed,  ComponentFixture } from '@angular/core/testing';
+import { Location } from '@angular/common';
+import { MdSnackBar, MdDialog } from '@angular/material';
+import { Router, ActivatedRoute } from '@angular/router';
 
 // Load the implementations that should be tested
 import { CourseOfActionComponent } from './course-of-action.component';
+<<<<<<< HEAD
```
Comment (mushinlogit) [inline src/app/settings/stix-objects/course-of-actions/course-of-action/course-of-action.component.spec.ts]: Can you resolve these commit merge markers?

Comment (sonoct) [reply src/app/settings/stix-objects/course-of-actions/course-of-action/course-of-action.component.spec.ts]: This should be fix

File: src/app/settings/stix-objects/attack-patterns/attack-pattern-list/attack-pattern-list.component.spec.ts (hunk 1 above)
Comment (sonoct) [reply src/app/settings/stix-objects/attack-patterns/attack-pattern-list/attack-pattern-list.component.spec.ts]: This should be fix

**Pull request merged**